buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -17,6 +17,10 @@ object Dependencies {
   val awsSqs = "com.amazonaws:aws-java-sdk-sqs:1.12.520"
   val bouncycastle = "org.bouncycastle:bcprov-jdk15on:1.70"
   val bouncycastlePgp = "org.bouncycastle:bcpg-jdk15on:1.70"
+  val bucket4jCore = "com.bucket4j:bucket4j-core:8.5.0"
```

Comment:
FYI, `8.6.0` just came out.

Reply:
Indeed, it only had a fix in functionality this doesn't use (local in-memory buckets). I'm [planning on doing a fast follow](https://cash.slack.com/archives/C01M5M297T7/p1699292746150629?thread_ts=1698944488.920549&cid=C01M5M297T7) to `8.7.0` to get `UnifiedJedis` support, for better compatibility with #2997